src/utils/daos_hdlr.c

```diff
@@ -2295,7 +2294,14 @@ fs_copy_dir(struct cmd_args_s *ap,
 		const char *d_name;
 
 		/* walk source directory */
-		entry = file_readdir(ap, src_file_dfs, src_dir);
+		rc = file_readdir(ap, src_file_dfs, src_dir, &entry);
+		if (rc != 0) {
+			rc = daos_errno2der(rc);
+			DH_PERROR_DER(ap, rc, "Cannot read directory");
```

Comment:
Should this print the directory?
```suggestion
			DH_PERROR_DER(ap, rc, "Cannot read directory '%s'", src_dir);
```

Reply:
@daltonbohning well I can cast it to (char*)? Since the macro will only take that and src_dir is a DIR*

Reply:
@dsikich Oops. Didn't realize is was DIR*. Maybe this is okay then